doc/monitoring/metrics_reference.rst

```diff
@@ -708,3 +708,172 @@ Event loop tx thread information:
+        *   -   ``tnt_memtx_tx_tnx_system_max``
+            -   The maximum number of bytes allocated by internals per transaction.
+
+.. _metrics-reference-memtx_mvcc:
+
+MVCC
+~~~~
+
+``mvcc`` is responsible for the isolation of transactions.
+It detects conflicts and makes sure that tuples that are no longer in the space, but read by some transaction
+(or can be read) have not been deleted.
+
+``trackers`` - trackers that keep track of transaction reads.
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 25 75
+        :header-rows: 0
+
+        *   -   ``tnt_memtx_tx_mvcc_trackers_total``
+            -   Trackers are allocated in total (in bytes).
+        *   -   ``tnt_memtx_tx_mvcc_trackers_avg``
+            -   Average for all transactions (total / number of transactions).
+        *   -   ``tnt_memtx_tx_mvcc_trackers_max``
+            -   Maximum trackers allocated per transaction (in bytes).
+
+``conflict`` - allocated in case of transaction conflicts.
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 25 75
+        :header-rows: 0
+
+        *   -   ``tnt_memtx_tx_mvcc_conflicts_total``
+            -   Bytes allocated for conflicts in total.
+        *   -   ``tnt_memtx_tx_mvcc_conflicts_avg``
+            -   Average for all transactions (total / number of transactions).
+        *   -   ``tnt_memtx_tx_mvcc_conflicts_max``
+            -   Maximum bytes allocated for conflicts per transaction.
+
+~~~~~~
+Tuples
+~~~~~~
+
+Saved tuples are divided into 3 categories: ``used``, ``read_view``, ``tracking``.
+
+Each category has two metrics:
+- ``retained`` tuples - they are no longer in the index, but MVCC does not allow them to be removed.
+- ``stories`` - MVCC is based on the story mechanism, almost every tuple has a story.
+This is a separate metric because even the tuples that are in the index can have a story.
+So ``stories`` and ``retained`` need to be measured separately.
+
+
+``used`` - tuples that are used by active read-write transactions.
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 25 75
+        :header-rows: 0
+
+        *   -   ``tnt_memtx_tx_mvcc_tuples_used_stories_count``
+            -   Number of ``used`` tuples / number of stories.
+        *   -   ``tnt_memtx_tx_mvcc_tuples_used_stories_total``
+            -   Amount of bytes used by stories ``used`` tuples.
+
+        *   -   ``tnt_memtx_tx_mvcc_tuples_used_retained_count``
+            -   Number of retained ``used`` tuples / number of stories.
+        *   -   ``tnt_memtx_tx_mvcc_tuples_used_retained_total``
+            -   Amount of bytes used by retained ``used`` tuples.
+
+``read_view`` - tuples that are not used by active read-write transactions,
+but are used by read-only transactions (i.e. in read view).
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 25 75
+        :header-rows: 0
+
+        *   -   ``tnt_memtx_tx_mvcc_tuples_read_view_stories_count``
```

Comment:
Is this an overall count or current one?

Reply:
Prometheus will definitely dislike `_count` here (https://github.com/tarantool/metrics/issues/117)

Reply:
Add metric label `count`.